### ISSUE TITLE
Can't import NES No-Intro DAT

### ISSUE THREAD

**Issue by leo60228** [alucryd/oxyromon#34]
For the DAT `Nintendo - Nintendo Entertainment System (20220418-134141).dat` I get this error:
```
Processing "Nintendo - Nintendo Entertainment System (20220418-134141).dat"
Error: SimpleError { err: "Failed to parse DAT file, duplicate field `clrmamepro`" }
```
This field is in fact duplicated (`<clrmamepro header="No-Intro_NES.xml"/>` and `<clrmamepro forcenodump="required"/>`) but presumably should not lead to an error.

Deleting the `forcenodump` element works fine.

**Comment by alucryd** [comment]
Thanks for the report, I'll find a way to have serde parse these correctly.

**Comment by alucryd** [comment]
Fixed in e182bb5f1620dd1bcb855186d50f7323e3eddd3d. I'll try to release next week.

**Comment by alucryd** [comment]
Just released 0.11.0, please let me know if you have run into any other issue.
